gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -243,6 +251,24 @@ class {{ service.async_client_name }}:
 
         )
 
+        if CLIENT_LOGGING_SUPPORTED and _LOGGER.isEnabledFor(std_logging.DEBUG):  # pragma: NO COVER
+            # TODO: Make this performant when logging is not enabled
+
+            # TODO: Remove this condition once the minimum version of google-auth is 2.35.0
+            credential_info = getattr(self.transport._credentials, "get_cred_info", None)
+            if callable(credential_info):
+                credential_info = credential_info()
+
+            _LOGGER.debug(
+                "Created client `{{ service.meta.address.proto_package_versioned }}.{{ service.async_client_name }}`.",
+                extra = {
+                    "serviceName": "{{ service.meta.address.proto }}",
+                    "universeDomain": getattr(self._client._transport._credentials, "universe_domain", ""),
+                    "credentialType": f"{type(self._client._transport._credentials).__module__}.{type(self._client._transport._credentials).__qualname__}",
+                    "credentialInfo": credential_info,
```

Comment:
Does `credential_info` contain actual secrets? In that case, I would use as the value here `f"[hash:{hash(credential_info}]"` or probably even better, `f"[hash:{hashlib.sha256(credential_info}]"`

Reply:
`credentials_info` was added here: https://github.com/googleapis/google-auth-library-python/commit/6f75dd5de9ee1da4509306ff2e6420b3d88f9d00.

It looks like it can possibly contain a source (path), type, and principal (email). So we're okay with not hashing it?